composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "andywer/js-localization",
+    "name": "jamoliddin-firdavs/js-localization",
```

Comment:
This should not change, it will be reflected in Packagist and this version won't be accessible with the old `andywer/js-localization` name